docs/core/integration-quality-scale/rules/iqs019.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "IQS019 - Can be discovered"
+---
+
+IQS019 is as follows:
+
+Can be discovered
```

Comment:
I renamed the rule to move the "if possible" to the exception. Frenck once mentioned we could do way more DHCP discovery, and looking at the numbers, I think I agree. So I'd rather have people go that extra mile and have their mac address checked, than to have them to uncheck it as "cant be discovered"